### ISSUE TITLE
add support for RTX Video HDR

### ISSUE THREAD

**Issue by Cxsmo-ai** [TheJollyBeardo/OBS_RTX_SuperResolution#2]
add support for RTX Video HDR

**Comment by Bemjo** [comment]
As of this post, this feature is currently locked as a driver feature for RTX video cards that cannot be utilized outside of what nvidia has enabled it for.

As soon as anything is available in an SDK, I will consider looking into how this fits into this plugin.

This feature would be at odds with what this plugin specifically does, and if I took the route of including it, I may have to rename this plugin entirely.

I may also consider making a new plugin just for this feature, however, I do not have any HDR displays and as such I do not consume proper HDR content at all so I'm really not sure If I'm the one to do anything about including RTX Video HDR support for OBS.

TLDR; I'll consider it when it's available, but no promises as it might just not fit.

**Comment by Bemjo** [comment]
Closing as nVidia has made no indication that this functionality would be coming to any SDK, and they have this functionality available through their nVidia App https://www.nvidia.com/en-us/software/nvidia-app/

The effects is applied as an overlay on-top of the game, and as such game captures can enable/disable the effect by enabling/disabling capture overlays. Display and Window captures were untested.

I have not done testing to see whether you can apply this effect to OBS itself and any projection windows you may use to display output from a video capture device through that (like trying to capture game console video), without ruining the entirety of your actual OBS capture if you were to stream/record that content.

Feel free to try for yourselves.
